src/test/java/org/reaktivity/k3po/nukleus/ext/SimplexIT.java

```diff
@@ -176,6 +177,17 @@ public void shouldThrottleInitialOnlyClientSentData() throws Exception
         k3po.finish();
     }
 
+    @Ignore ("Please see issue #1206: https://github.com/kaazing/tickets/issues/1206")
```

Comment:
Please replace this bug with one filed against reaktivity/k3po-nukleus-ext, as I requested in [the bug](https://github.com/kaazing/tickets/issues/1206), and update this link point to the new one.